cerberus/prometheus/client.py

```diff
@@ -23,3 +24,25 @@ def get_metrics(query):
         return prom_cli.custom_query(query=query, params=None)
     except Exception as e:
         logging.error("Failed to get the metrics: %s" % e)
+
+
+# Alert on high latencies
+def process_prom_query(distribution, prometheus_url, prometheus_bearer_token, query):
+    # Intialize prometheus client
+    if distribution == "openshift" and not prometheus_url:
+        url = runcommand.invoke(r"""oc get routes -n openshift-monitoring -o=jsonpath='{.items[?(@.metadata.name=="prometheus-k8s")].spec.host}'""") # noqa
+        prometheus_url = "https://" + url
+    if distribution == "openshift" and not prometheus_bearer_token:
+        prometheus_bearer_token = runcommand.invoke("oc -n openshift-monitoring "
+                                                    "sa get-token prometheus-k8s")
+    if prometheus_url and prometheus_bearer_token:
+        initialize_prom_client(prometheus_url, prometheus_bearer_token)
+        metrics = get_metrics(query)
+        if metrics:
+            logging.warning("Kubernetes API server latency is high. "
```

Comment:
We might want to move this log to start_cerberus.py script as it's specific to the query.

Reply:
Fixed it.